policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/income_limit.yaml

```diff
@@ -52,10 +152,14 @@ metadata:
     # Income threshold is not specified in the legal code
     - title: §5219-KK. Property tax fairness credit for tax 
       href: https://legislature.maine.gov/statutes/36/title36sec5219-KK.html
```

Comment:
do these values align with the 6% in the legal code? if so refactor to use that

Reply:
They don't align, but they do interact. We use the income limit* 6% to determine eligibility.

Reply:
might this be derived from the phase-out schedule instead then? an income limit would typically be too important for lawmakers to leave to the department of revenue to decide. please check if these numbers relate mathematically to other parameters and whether the condition actually binds.